core/enumerable/uniq_spec.rb

```diff
@@ -19,8 +19,16 @@ def each
         end
       end
 
-      it 'returns an array that contains only first argument of the yield' do
-        @enum.uniq { |_, label| label.downcase }.should == [0, 2]
+      ruby_version_is '2.5' do
+        it 'returns all yield arguments as an array' do
+          @enum.uniq { |_, label| label.downcase }.should == [[0, 'foo'], [2, 'bar']]
+        end
+      end
+
+      ruby_bug '#13669', ''...'2.5' do
```

Comment:
`''...'2.5'` means excluding 2.5, `'2.5'` means until 2.5 inclusive.
ruby-head in Travis is old (from May), that's why it fails.